conans/client/generators/pkg_config.py

```diff
@@ -67,6 +67,7 @@ def content(self):
     def _pc_file_content(self, name, cpp_info, requires_gennames):
         prefix_path = cpp_info.rootpath.replace("\\", "/")
         lines = ['prefix=%s' % prefix_path]
+        quote = " " in prefix_path
```

Comment:
can't quotes be added always? trying to avoid some risks?

Reply:
yes, I was thinking it's very risky change.

Reply:
changed to always add quotes...